internal/configuration/validator/configuration.go

```diff
@@ -24,6 +24,12 @@ func Validate(configuration *schema.Configuration, validator *schema.StructValid
 		configuration.LogsLevel = defaultLogsLevel
 	}
 
+	if configuration.SSLKey != "" && configuration.SSLCert == "" {
+		validator.Push(fmt.Errorf("SSL key provided but no certificate"))
```

Comment:
I feel like we could be much more concise in our messaging here. It doesn't matter if they have done something correct we should explicitly be calling out what wasn't done and potentially is causing issues.

```suggestion
		validator.Push(fmt.Errorf("No SSL certificate provided, please check the "ssl_cert" which has been configured"))
```